show/main.py

```diff
@@ -15,9 +15,15 @@
 import sonic_platform
 from swsssdk import ConfigDBConnector
 from swsssdk import SonicV2Connector
+from portconfig import get_port_config
 
 import mlnx
 
+PLATFORM_ROOT_PATH = '/usr/share/sonic/device'
+SONIC_CFGGEN_PATH = '/usr/local/bin/sonic-cfggen'
+HWSKU_KEY = 'DEVICE_METADATA.localhost.hwsku'
+PLATFORM_KEY = 'DEVICE_METADATA.localhost.platform'
```

Comment:
Why add these four constants? They are not used anywhere in the file.

Reply:
Other than SONIC_CFGGEN_PATH removed unused constants